services/block.go

```diff
@@ -21,6 +21,12 @@ const OpBurn = "Burn"
 // OpStatusOK is the OK status.
 const OpStatusOK = "OK"
 
+// SupportedOperationTypes is a list of the supported operations.
+var SupportedOperationTypes = []string{
+	OpTransfer,
+	OpBurn,
+}
+
```

Comment:
This was written in preparation for the next rosetta-sdk-go bump, which will require this as an argument to the Asserter.